File: app/src/main/java/com/v2/desafionubank/ui/view/ActionViewHolder.java
package com.v2.desafionubank.ui.view;

import android.view.View;
import android.widget.TextView;

import com.v2.desafionubank.R;
import com.v2.desafionubank.model.NoticeAction;

import butterknife.BindView;

/**
 * Created by CaioSChristino on 10/04/18.
 */

public class ActionViewHolder extends BaseViewHolder {
    @BindView(R.id.title_item)
    TextView title;

    public ActionViewHolder(View view) {
        super(view);
    }

    public void setTitle(NoticeAction item) {
        this.title.setText(item.getTitle().toUpperCase());
        if (item.getAction().equals("continue")) {
            this.title.setTextColor(itemView.getContext().getResources().getColor(R.color.colorEnablePurple));
        }
    }

    public void setOnclick(View.OnClickListener onclick) {
        itemView.setOnClickListener(onclick);
    }
}

File: app/src/main/java/com/v2/desafionubank/ui/view/ActionViewHolder.kt
package com.v2.desafionubank.ui.view

import android.view.View
import android.widget.TextView
import butterknife.BindView
import com.v2.desafionubank.R
import com.v2.desafionubank.model.NoticeAction

/**
 * Created by csanchez on 19/04/2018.
 */

class ActionViewHolder(view: View) : BaseViewHolder(view) {
    @BindView(R.id.title_item)
    internal var title: TextView? = null

    fun setTitle(item: NoticeAction) {
        this.title!!.text = item.title!!.toUpperCase()
        if (item.action == "continue") {
            this.title!!.setTextColor(itemView.context.resources.getColor(R.color.colorEnablePurple))
        }
    }

    fun setOnclick(onclick: View.OnClickListener) {
        itemView.setOnClickListener(onclick)
    }
}